kernels/revm/utils/src/crypto/sbi.rs

```diff
@@ -21,9 +22,36 @@ pub const SBI_TEZOS_KECCAK256_HASH: u64 = 0x0b;
 /// To limit size of proofs in refutation games
 pub const MAX_PVM_MEMORY_ACCESS: usize = 4096;
 
+pub fn batch_verify(txs: &[SignedOperation]) -> bool {
+    let verification_data: Vec<_> = txs
```

Comment:
At a minimum, it would be good to define a `pub type SigVerificationBatch = Vec<([u8; 65], [u8; 64], [u8; 32])>;` both here and in `tezos.rs` and use it on both sides.

```suggestion
    let verification_data: Vec<SigVerificationBatch> = txs
```

Reply:
addressed this